src/ui/components/LedgerAdmin.js

```diff
@@ -1,22 +1,22 @@
 // @flow
 
-import React, {useState, type Node as ReactNode} from "react";
+import React, {useState, useMemo, type Node as ReactNode} from "react";
 import {makeStyles} from "@material-ui/core/styles";
 import ButtonGroup from "@material-ui/core/ButtonGroup";
 import {
   Button,
   Checkbox,
   Container,
-  Divider,
   FormControlLabel,
   List,
-  ListItem,
   TextField,
 } from "@material-ui/core";
 import {useLedger} from "../utils/LedgerContext";
+import {useTableState} from "../../webutil/tableState";
 import {IdentityMerger} from "./IdentityMerger";
 import {type Identity, type IdentityId} from "../../core/identity";
 import {AliasView} from "./AliasView";
```

Comment:
Is the AliasView component where the unstyled Aliases links are?

Reply:
Why yes, yes it is.